### ISSUE TITLE
Right-Click, show in finder

### ISSUE THREAD

**Issue by tdlivings** [dscottbuch/cTiVo#278]
Right click, show in finder not working even though directory is valid and file exists.

**Comment by mackworth** [comment]
So, not seeing that on mine.
Is this from the Now Playing list or the Download list? 
And the "Show in Finder" command is enabled (aka not greyed out), it just doesn't work? 
Does it switch to Finder, or just leave you were? 
Is the show in the Now Playing list in bold face?
Do other downloaded shows do the same thing?

**Comment by tdlivings** [comment]
Yes from the now playing and yes it is greyed out. It doesn’t do anything on my machine. The Now playing doesn’t reflect downloaded as the 2.5 “bold”. All show do the same. 

It might be important to say, on previous versions, When I download from Machine A and sync to machine b, The cTivo Now Playing senses the downloaded content resulting in bold line items on both A and B. With the new version Machine A shows bold but Machine B does not.

Now I am unable tp download. CTIVO reports disk is full but I have 1.65 TB left. Tried 2 different disks.

**Comment by tdlivings** [comment]
If I change tmp location the disk full message doesn't appear and I can download once again.

**Comment by tdlivings** [comment]
Must have been just one of those things, re-entered tmp/ctivo and download works but original it still there.

**Comment by mackworth** [comment]
There's apparently some cases where the OS does not provide attributes (such as disk space) for a volume. Beta2 now checks for that and allow the download if there is no info on diskspace.

So, how are the file shared between Machine A and B?  And does it still work in 2.5.1 but not in beta2?  This would be hard to do.

For background, we use the Spotlight comment field to store a comment "cTiVoDownload".  If you go to the Finder, and do a search, then hit Other in pulldown menu, it will show you all the search attributes. Go to Spotlight Comment, and turn on the "In Menu" field. Now pulldown the menu and select Spotlight Comment, then put cTiVoDownload into the search field. You should see all your movies on disk.  If you see them on Machine A, but don't see them on Machine B, then your sync process isn't maintaining the metadata properly.

Second part is that after finding a downloaded file, we look for two attributes in the metadata: TiVoName and TiVoID to match it up to TiVo's list. Don't think there's any way to see these in Finder.  If you go into Terminal and type `xattr -l`, then drag/drop the file onto the Terminal window and hit Return, it will show you the three attributes.  Again, if not copied over properly, it won't work.

**Comment by tdlivings** [comment]
All good information. All I know without digging deeper is it worked in Sierra and not High Sierra. The results before were the ability to drag from now playing list directly into a video editor. Since these mapping’s are no longer set this feature is no longer available.  What I had before was bold on any entry listed on any machine that had the valid content directory mapped. No doubt much has changed since. I did a search and found creation date for all Tivo files to be 1969. I found this to be odd. 

Still, the ctivo tool is still a lifesaver. No complaints here. I will continue to dig in

Thank you
